lib/ansible/modules/database/mysql/mysql_variables.py

```diff
@@ -32,6 +32,12 @@
         description:
             - If set, then sets variable value to this
         type: str
+    persist:
+        description:
+            - If true, sets variable persistently (will survive mysql restarts)
```

Comment:
```suggestion
            - If C(true), sets variable persistently (will survive mysql restarts). Supported from MySQL 8.0.
```
about version, needs to be checked